janus/lib/server.rb

```diff
@@ -23,6 +23,13 @@ class Server < Etna::Server
 
     get '/project/:project_name', action: 'admin#project', auth: { user: { can_edit?: :project_name } }
 
+    get '/refresh_token', action: 'user#refresh_token', auth: { user: { active?: true } }
+
+    # Once we figure out a long-term token strategy, this should probably get
+    #   consolidated with /refresh_token so we don't just keep creating
+    #   small token-related views.
+    get '/viewer_token', action: 'user#viewer_token', auth: { user: { is_superuser?: true } }
```

Comment:
I've kept these endpoints separate for now, since I think the `viewer_token` will only be used by us for testing. If other users also need it, I can merge this route into the `refresh_token`.